PoorMansTSqlFormatterLibShared/ObfuscatingKeywordMapping.cs

```diff
@@ -34,7 +34,7 @@ static ObfuscatingKeywordMapping()
             Instance.Add("LEFT OUTER JOIN", "LEFT JOIN");
             Instance.Add("RIGHT OUTER JOIN", "RIGHT JOIN");
             Instance.Add("FULL OUTER JOIN", "FULL JOIN");
-            Instance.Add("INNER JOIN", "JOIN");
+            //Instance.Add("INNER JOIN", "JOIN");
```

Comment:
Disabling this line stops the formatter from converting `JOIN` to `INNER JOIN`, right?

If that's how this works most of the entries in this file look ok, but does that mean it also converts `VARCHAR` to `CHAR VARYING` and `INT` to `INTEGER`?

Reply:
I reverted this change as I found that these mappings are only utilized when the KeywordStandardization option of the formatter is set to true